README.md

```diff
@@ -31,3 +31,68 @@ There are capistrano tasks to connect to the Solr UI for managing solr that can
 
 * Would it be better to go from a server on Princeton's network (e.g., the bibdata-staging worker machine), so that Developer's internet connection doesn't affect benchmarking
 
+## Running from loadtest.lib.princeton.edu
+* Ensure the User Defined Variables in the Solr test plan are correct and saved for the environment
+* Copy the test file onto the remote host
+```bash
+scp solr_test_plan.jmx deploy@loadtest.lib.princeton.edu:~/
+```
+* Copy the keywords file onto the remote host
+```bash
+scp keywords.csv deploy@loadtest.lib.princeton.edu:~/
+```
+* SSH onto the box
+```bash
+ssh deploy@loadtest.lib.princeton.edu
+```
+* Run the jmeter test from loadtest.lib.princeton.edu
+```bash
+jmeter -n -t solr_test_plan.jmx
+```
+* Copy the results to your local machine (from the local)
+```bash
+scp deploy@loadtest.lib.princeton.edu:~/test_results/simple_data_writer.csv ./test_results
+```
+## User Defined Variables by Environment
+### Development
+host orangelight.dev.solr.lndo.site
```

Comment:
The formatting on this is running everything together. Should it be a list?

Reply:
Ah, thanks, I'll put it in a code block.